### PR TITLE
Update Babel and Traceur.

### LINKED ISSUE #441: Update Babel and Traceur
In the package.jsons and bower.jsons.  Hopefully there is no breaking changes.

### PR DESCRIPTION
**Pull request by m-mujica**
Closes #441.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "grunt-simple-mocha": "^0.4.0",
     "testee": "0.1.1",
     "jquery": ">2.0",
-    "babel-core": "~4.1.1"
+    "babel-core": "~5.8.14"
```
Comment (matthewp) [inline package.json]: can we pick this to 5.8.14 (remove the ~).  The reason is otherwise it is easy to accidentally check in babel-core when you run "grunt build" for unrelated changes.

**Comment by m-mujica** [comment]
@matthewp done! I did the same in Transpile.
